src/components/OnboardingMenu/OnboardingMenu.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+
+import {Button, Icon, IconData, Progress} from '@gravity-ui/uikit';
+
+import {block} from '../utils/cn';
+
+import i18n from './i18n';
+
+import './OnboardingMenu.scss';
+
+const cnOnboardingMenu = block('guide');
+
+export type OnboardingMenuProps = {
+    title: React.ReactNode;
+    progress: number;
+    icon?: IconData;
+
+    defaultExpanded: boolean;
+    children?: React.ReactNode;
+
+    collapseButtonText?: string;
+    completeButtonText?: string;
+    className?: string;
+
+    onExpand?: (expanded: boolean) => void;
+    onCompleteClick: () => void;
+    onRollUpClick?: (event: React.MouseEvent) => void;
+};
+
+export const OnboardingMenu = ({
```

Comment:
You must also reexport all of the public components in the `src/components/OnboardingMenu/index.ts` file, and also add reexport to the `src/components/index.ts`